.ci/.jenkins-agents.yml

```
@@ -17,3 +17,5 @@ agents:
   - REPO: "apm-agent-ruby"
     FEATURES_PATH: "features"
     JSON_SPECS_PATH: "spec/fixtures"
+  - REPO: "apm-agent-nodejs"
+    JSON_SPECS_PATH: "tests/fixtures/json-specs"
```

```suggestion
    JSON_SPECS_PATH: "test/fixtures/json-specs"
```

We want "test" singular probably, right?

---

🤦 fixed https://github.com/elastic/apm/pull/366/files#diff-c76f60989ba64bf6dd904239fdb2a8562bbd3e6a965cf69cc59ebc595a480142R21